### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by IbrahimElkhatib**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# test
+# test project
```
Comment (mustafahawi) [inline README.md]: Project

Comment (mustafahawi) [inline README.md]: Test

**Review by mustafahawi — CHANGES_REQUESTED** [review]
Fix Readme message to Test Project

File: README.md (hunk 1 above)
Comment (IbrahimElkhatib) [reply README.md]: Ok

Comment (IbrahimElkhatib) [reply README.md]: Done

**Review by IbrahimElkhatib — COMMENTED** [review]
Done

**Review by mustafahawi — APPROVED** [review]

**Review by mustafahawi — APPROVED** [review]

**Review by mustafahawi — APPROVED** [review]
Ok

**Review by mustafahawi — APPROVED** [review]

**Pull request merged**